test/geocoders/arcgis.py

```diff
@@ -63,6 +63,34 @@ def test_unicode_name(self):
             {"latitude": 39.916, "longitude": 116.390},
         )
 
+    def test_geocode_with_out_fields_string(self):
+        """
+        ArcGIS.geocode with outFields string
+        """
+        self.geocode_run(
+            {"query": "Heathrow Airport", "out_fields":"Country"},
+            {"raw": {
+                u'attributes': {u'Country': u'GBR'},
+                u'score': 100, u'location': {u'y': 51.47114986100047, u'x': -0.45648655399958216},
+                u'extent': {u'xmin': -0.481488, u'ymin': 51.44615, u'ymax': 51.49615, u'xmax': -0.431488},
```

Comment:
It might be a bad idea to test the exact coordinates/extent of the airport, since those might change if the expansion plans go ahead :)